aspnetcore/signalr/webpack-and-typescript.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Using ASP.NET Core SignalR with TypeScript and WebPack
+author: ssougnez
+description: How to bundle a web application using SignalR, TypeScript and WebPack.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: ssougnez
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/webpack-and-typescript
+---
+
+# Using ASP.NET Core SignalR with TypeScript and WebPack
+
+By [Sébastien Sougnez](https://twitter.com/ssougnez)
+
+**WebPack** enables developers to easily bundle the client side of a web application. This tutorial teaches how to use it to bundle an application written in **TypeScript** that communicates with a server through web sockets using **SignalR**.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/webpack-and-typescript/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Prerequisites
+
+* **[NodeJs](https://nodejs.org/en/)** and **npm** must be installed on your system to be able to use **WebPack**.
```

Comment:
- NodeJs --> Node.js
- Remove "en/" from the URL. The browser will determine the appropriate locale.